### ISSUE TITLE
SYNTHESIZER: Rethinking Self-Attention in Transformer Models 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#89]
# TL;DR
This study revisits the Self-Attention of a Transformer. Self-Attention uses dot-products to take interactions between tokens. In this study, they calculate the attention weight of each token independently or treat attention weight as a training parameter instead of dot-product, but they show the performance is competitive.

![image](https://user-images.githubusercontent.com/24524018/92561198-b4d04200-f2ae-11ea-921e-73ebb65a21af.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2005.00743

# Submission Dates(yyyy/mm/dd)

# Authors and institutions

# Methods

# Results

# Comments
